prover/prover_fri_gateway/src/main.rs

```diff
@@ -22,8 +28,17 @@ mod proof_submitter;
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
-    let observability_config =
-        ObservabilityConfig::from_env().context("ObservabilityConfig::from_env()")?;
+    let opt = Cli::parse();
+    let observability_config = match opt.observability_config_path {
+        Some(path) => {
+            let yaml =
+                std::fs::read_to_string(path).context("Failed to read observability config")?;
+            decode_yaml_repr::<Observability>(&yaml)
+                .context("Failed to parse observability config")?
+        }
+        None => ObservabilityConfig::from_env().context("ObservabilityConfig::from_env()")?,
+    };
```

Comment:
Nope, it's all in general config

Reply:
we have only this set of files 
https://github.com/matter-labs/zksync-era/tree/main/etc/env/file_based